### ISSUE TITLE
tibialyzer 1.2.2 (beta) not working on chrome

### ISSUE THREAD

**Issue by CladesAffero** [Mytherin/Tibialyzer#89]
i wanted to use stable release but only worked on firefox and i read 1.2.2 supported chrome. however it still doesn't detect it or works properly but it does show green on status indicator on the top left corner

**Comment by TheLateTes** [comment]
I've personally been using 1.2.2 (beta) on chrome since it was released. While my experience hasn't been what I consider perfect it is very functional. When you have it open are you pushing the detect flash client button on the Main tab under Global Settings?

**Comment by CladesAffero** [comment]
yes i am, when i click on detect status bar becomes green but no flash
client is selectable in client selection neither works just clicking on
detect flash client. i'll try it again tomorrow maybe i messed up something

**Comment by SartoriRafael** [comment]
For me It works only in 32-bit chrome, not 100% but it works. In the 64-bit version does not work.

**Comment by CladesAffero** [comment]
yeah i tried again and didn't work. i'm using 64-bit so maybe that's why
